### DISCUSSION TITLE
Welcome to behave Discussions!

### DISCUSSION THREAD

**Discussion by jenisys** [behave/behave#882] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bittner** [comment]
Seriously? I need to introduce myself!

> My name is Peter. I have a wife, 3 kids and 5 rabbits. I love programming, preferably in Python.

Voilà!

**Reply by jenisys** [reply]
Mmh, I just enabled the github/discussion functionality and used the default template without really having too much time to read over  it, maybe I better should. Nice introduction anyway, @bittner.
